js/bitfinex2.js

```diff
@@ -435,19 +435,55 @@ module.exports = class bitfinex2 extends bitfinex {
         throw new NotSupported (this.id + ' withdraw not implemented yet');
     }
 
-    async fetchMyTrades (symbol = undefined, since = undefined, limit = 25, params = {}) {
+    async fetchMyTrades (symbol = undefined, since = undefined, limit = 1000, params = {}) {
         await this.loadMarkets ();
         let market = this.market (symbol);
         let request = {
             'symbol': market['id'],
             'limit': limit,
-            'end': this.seconds (),
+            'end': this.milliseconds (),
         };
         if (since !== undefined)
-            request['start'] = parseInt (since / 1000);
+            request['start'] = since.getTime ();
```

Comment:
We can't use getTime() here, because it's JS-specific. Since is in milliseconds already, according to the Manual.

Reply:
ok thanks.